apps/api/src/app/app.module.ts

```diff
@@ -31,11 +31,7 @@ import { FeedbackModule } from '../feedback/feedback.module'
     ConfigModule.forRoot({
       isGlobal: true,
       envFilePath: '../../.env',
-      validate: EnvSchema.parse,
-      validationOptions: {
-        allowUnknown: false,
```

Comment:
The validation options was part of the joi library which is supported by nest js inbuilt. but since we aint using joi, its better to remove the `validationOptions` since it doesn't really do anything.

Reply:
okay!